packages/e2e-test-utils/src/click-block-toolbar-button.js

```diff
@@ -6,15 +6,25 @@ import { showBlockToolbar } from './show-block-toolbar';
 /**
  * Clicks a block toolbar button.
  *
- * @param {string} buttonAriaLabel The aria label of the button to click.
+ * @param {string} content The text string of the button label or content.
+ * @param {string} type The type of button - 'ariaLabel' or 'text'.
  */
-export async function clickBlockToolbarButton( buttonAriaLabel ) {
+export async function clickBlockToolbarButton( content, type = 'ariaLabel' ) {
```

Comment:
I meant to change the `text` type value to `content`, not the `type` parameter. I assume that was just an oversight, so I'll push a commit to fix that and then merge.

Reply:
woops  - sorry, thanks for changing, I just [updated the tests](https://github.com/WordPress/gutenberg/pull/23704/files#diff-2c145aff524ea38590e643f0fddc599cR86-R97l) to match this change as well.